src/utils/seo.ts

```diff
@@ -29,6 +29,11 @@ type CompanyInfo = {
   defaultSEO: SeoData;
```

Comment:
not sure this is appropriate to do in this PR, but I fear the seo.ts has a lot of complexity now.. I wonder if we should just take a quick look if we have time this week? Say. if you're closing in on finishing the SEO target goal, we can cleanup a little?

Reply:
:+1: I have defined a separate issue for this: #604